LoopUI/Charts/PredictedGlucoseChart.swift

```diff
@@ -66,11 +67,19 @@ public class PredictedGlucoseChart: GlucoseChart, ChartProviding {
 
     public private(set) var endDate: Date?
 
+    private var predictedGlucoseSuggestedBounds: PredictedGlucoseBounds?
```

Comment:
[nit] Consider calling just `predictedGlucoseBounds` since that is how you use it all below.

Reply:
hm... they are the suggested bounds and can be overridden when a value is above or below those bounds. 

To make it clearer, I'll update to `predictedGlucoseSoftBounds`.